src/components/TaskCard/index.jsx

```diff
@@ -1,30 +1,68 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import PropTypes from "prop-types";
+import classNames from "classnames";
 import { useDispatch } from "react-redux";
-import { deleteTask } from "store/actions";
-import { ICON_DELETE, DELETE_ICON_ALT_TEXT } from "utils/constant/images";
+import { deleteTask, completeTask } from "store/actions";
+import {
+  ICON_COMPLETE,
+  COMPLETE_ICON_ALT_TEXT,
+  ICON_DELETE,
+  DELETE_ICON_ALT_TEXT,
+} from "utils/constant";
+import { getDateDifference } from "utils/helpers/getDateDifference";
 import "components/TaskCard/index.scss";
 import { checkDateString } from "utils/helpers/propCustomValidation";
 import { dateFormatter } from "utils/helpers/dateFormatter";
 
-const TaskCard = ({ id, task, createdTime }) => {
+const TaskCard = ({ id, task, createdTime, completed }) => {
+  const [taskCompletedIn, setTaskCompletedIn] = useState(null);
+
+  const taskTextStyle = classNames({
+    "task-card__task": true,
+    "task-card__task--done": completed,
+  });
+
+  useEffect(() => {
+    setTaskCompletedIn(getDateDifference(createdTime));
+  }, []);
+
   const dispatch = useDispatch();
 
+  function handleCompleteTask() {
+    dispatch(completeTask(id));
+  }
+
   function handleDeleteTask() {
     dispatch(deleteTask(id));
   }
 
+  const completedInText = `Completed in ${taskCompletedIn} ${
+    taskCompletedIn > 1 ? "days" : "day"
+  }`;
```

Comment:
consider the following refactoring which improves code readability.

```
  function TaskCompleteDays({ days }) (
      <p className="task-card__completed">Completed in {days} `${days > 1 ? "days" : "day"}</p>
  );
```

```suggestion
<TaskCompleteDays days={taskCompletedIn}  />
```